docs/DeveloperGuide.md

```diff
@@ -671,58 +670,62 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
     * Steps 1a1-1a2 are repeated until the number entered is valid.
     * Use case resumes from step 2.
 
-**Use case: UC8 - Clear the displayed list**
+**Use case: UC8 - Sort the displayed list**
 
 **MSS**
 
-1. User requests clear the displayed list.
-1. EmployeeManager clears the displayed list.
-1. EmployeeManager displays an empty list.
+1. User requests to sort the displayed list based on an attribute.
+1. EmployeeManager displays the sorted list based on the specified attribute.
 1. Use case ends.
 
-**Use case: UC9 - Exporting the data file**
+**Extensions**
+
+* 1a. The user enters an invalid attribute.
+    * 1a1. EmployeeManager shows an error message.
+
+  Use case ends.
+
+
+* 1b. The user enters multiple attributes.
+    * 1b1. EmployeeManager shows an error message.
```

Comment:
Right now our application will sort based on the first attribute given right instead of showing an error message?

Reply:
It shows an error message for me